can_tools/scrapers/official/ME/me_vaccines.py

```diff
@@ -185,3 +186,382 @@ def normalize(self, resjson: dict) -> pd.DataFrame:
         out = self._reshape_variables(df, variables)
         out["dt"] = self._retrieve_dt("US/Eastern")
         return out
+
+
+class MaineRaceVaccines(MicrosoftBIDashboard):
+    has_location = False
+    location_type = "county"
+    state_fips = int(us.states.lookup("Maine").fips)
+
+    source = "https://www.maine.gov/covid19/vaccines/dashboard"
+    source_name = "Covid-19 Response Office of the Governor"
+    powerbi_url = "https://wabi-us-east-a-primary-api.analysis.windows.net"
+
+    demographic = "race"
+    demographic_query_name = "Race"
+
+    variables = {
+        "initiated_total": v.INITIATING_VACCINATIONS_ALL,
+        "complete_total": v.FULLY_VACCINATED_ALL,
+    }
+
+    col_mapping = {
+        "G0": "county",
+        "M_1_DM3_{demo}_M_0_DM4_0_C_0": "first_dose_total",
+        "M_1_DM3_{demo}_M_0_DM4_0_C_1": "complete_total",
+        "M_1_DM3_{demo}_M_1_DM5_0_C_2": "jj_complete",
```

Comment:
@smcclure17 what is the meaning of `jj_complete` and `jj_init`? Are they always the same?

Reply:
they seem to store the J&J doses in one of two locations. Each manufacturer returns dose-specific data in the JSON. For the pfizer and moderna vaccines, the first/second doses are stored in a list (inside a dict/object) where the first index is the # of first doses and the second index is the # of second doses. For the J&J doses, they seem to put them arbitrarily in either index (either the 'first dose' or 'second dose' index)

Here's what the data looks like when scraped:
![image](https://user-images.githubusercontent.com/55333380/120059189-e3996e80-c01d-11eb-8fbd-33a64146eab0.png)

Reply:
That makes sense -- good sleuthing and figuring this out. I think we are good to merge!